webscrapers/BCB_IF/classes/BancoCentral.py

```diff
@@ -0,0 +1,104 @@
+# Quantitative Finance
+# Insper
+# Authors: Ivan Zemella and Renan Pereira
+# Class
+
+import json,requests
+
+class BancoCentral:
+	
+	url_bancos 	= "https://www3.bcb.gov.br/ifdata/rest/arquivos?nomeArquivo=__trimestre__/cadastro__trimestre___1005.json&{}"
+	
+	url_valores = "https://www3.bcb.gov.br/ifdata/rest/arquivos?nomeArquivo=__trimestre__/dados__trimestre___1.json&{}"
+	
+	indicador_patrimonio_liquido = 78186
+	
+	indicador_lucro_liquido = 78187
```

Comment:
The indicator code should also be an user input. That way **any** code can be acquired.